hamocc/mo_hamocc4bcm.F90

```diff
@@ -90,7 +90,7 @@ subroutine hamocc4bcm(kpie,kpje,kpke,kbnd,kplyear,kplmon,kplday,kldtday,pdlxp,pd
     real,    intent(in)  :: prho   (kpie,kpje,kpke)                         ! density [g/cm^3].
     real,    intent(in)  :: pglat  (1-kbnd:kpie+kbnd,1-kbnd:kpje+kbnd)      ! latitude of grid cells [deg north].
     real,    intent(in)  :: omask  (kpie,kpje)                              ! land/ocean mask.
-    real,    intent(in)  :: dust   (kpie,kpje)                              ! dust deposition flux [kg/m2/month].
+    real,    intent(in)  :: dust   (kpie,kpje,ndust)                        ! dust/sFe deposition flux [kg/m2/month].
```

Comment:
As far as I understand, the units have changed here - please update the comment

Reply:
Done